NOTICES.md

```diff
@@ -0,0 +1,37 @@
+# Notices for OpenJCEPlus Module
+
+* Project home: https://github.com/IBM/OpenJCEPlus
+
+## Trademarks
+
+Java and all Java-based trademarks are trademarks of Oracle Corporation in the United States, other countries, or both.
+
+## Applicable Licenses
+
+Apache-2.0. See included module OPENJCEPLUS_LICENCE file for details.
+
+As part of a FIPS 140-3 technology preview capability, this Semeru Runtimes distribution includes a modified OpenSSL 1.1.1 toolkit binary that is currently undergoing FIPS 140-3 certification testing at the U.S. National Institute of Standards and Technology (NIST). Because this toolkit version was built and submitted to NIST before we contributed its code to an open source project (now at https://github.com/IBM/OpenCryptographyKitC), the toolkit binary included in this release contains strings that reference an IBM license. These strings have been removed from the open-source version of the toolkit.
```

Comment:
is this supposed to be one lone line?

Reply:
The file is in Markdown format similar to other notices i saw in the legal directory. I could add newlines here in HTML format but not sure how that will show up in the final legal directory. Id assume the file would end up having HTML embedded. Seems new lines vs not using new lines is controversial according to this.

https://www.markdownguide.org/basic-syntax/#line-break-best-practices

Reply:
I added newlines to the file. If viewable as a text file this will certainly look better using https://github.com/ibmruntimes/OpenJCEPlus/pull/3/commits/ab03e39ea913c49ed099144bdb5ed664b20da1af

Reply:
To further clarify i added text newlines here not HTML line breaks.

Reply:
there is a missing phrase that I think was dropped in an early draft and should have been added back. The last sentence should read "These strings have been removed from the open-source version of the toolkit in addition to other minor updates."

Reply:
Updated this last sentence using https://github.com/ibmruntimes/OpenJCEPlus/pull/3/commits/83746b19cfbe75e0c346e22240515e4aeb8fe56a this adds the wording ` in addition to other
minor updates` to the sentence.